rocketpy/rocket/aero_surface/linear_generic_surface.py

```diff
@@ -0,0 +1,405 @@
+                * beta
+            )
+
+        return Function(
+            total_coefficient,
+            [
+                'alpha',
+                'beta',
+                'mach',
+                'reynolds',
+                'pitch_rate',
+                'yaw_rate',
+                'roll_rate',
+            ],
+            ['coefficient'],
+        )
+
+    def compute_damping_coefficient(self, c_p, c_q, c_r):
+        """Compute the damping coefficient from the derivatives of the
+        aerodynamic coefficients."""
+
+        def total_coefficient(
+            alpha, beta, mach, reynolds, pitch_rate, yaw_rate, roll_rate
+        ):
+            return (
+                c_p(alpha, beta, mach, reynolds, pitch_rate, yaw_rate, roll_rate)
+                * pitch_rate
+                + c_q(alpha, beta, mach, reynolds, pitch_rate, yaw_rate, roll_rate)
+                * yaw_rate
+                + c_r(alpha, beta, mach, reynolds, pitch_rate, yaw_rate, roll_rate)
+                * roll_rate
+            )
+
+        return Function(
+            total_coefficient,
+            [
+                'alpha',
+                'beta',
+                'mach',
+                'reynolds',
+                'pitch_rate',
+                'yaw_rate',
+                'roll_rate',
+            ],
+            ['coefficient'],
+        )
+
+    def compute_all_coefficients(self):
+        """Compute all the aerodynamic coefficients from the derivatives."""
+        # pylint: disable=invalid-name
+        self.cLf = self.compute_forcing_coefficient(
+            self.cL_0, self.cL_alpha, self.cL_beta
+        )
+        self.cLd = self.compute_damping_coefficient(self.cL_p, self.cL_q, self.cL_r)
+
+        self.cQf = self.compute_forcing_coefficient(
+            self.cQ_0, self.cQ_alpha, self.cQ_beta
+        )
+        self.cQd = self.compute_damping_coefficient(self.cQ_p, self.cQ_q, self.cQ_r)
+
+        self.cDf = self.compute_forcing_coefficient(
+            self.cD_0, self.cD_alpha, self.cD_beta
+        )
+        self.cDd = self.compute_damping_coefficient(self.cD_p, self.cD_q, self.cD_r)
+
+        self.cmf = self.compute_forcing_coefficient(
+            self.cm_0, self.cm_alpha, self.cm_beta
+        )
+        self.cmd = self.compute_damping_coefficient(self.cm_p, self.cm_q, self.cm_r)
+
+        self.cnf = self.compute_forcing_coefficient(
+            self.cn_0, self.cn_alpha, self.cn_beta
+        )
+        self.cnd = self.compute_damping_coefficient(self.cn_p, self.cn_q, self.cn_r)
+
+        self.clf = self.compute_forcing_coefficient(
+            self.cl_0, self.cl_alpha, self.cl_beta
+        )
+        self.cld = self.compute_damping_coefficient(self.cl_p, self.cl_q, self.cl_r)
+
+    def _compute_from_coefficients(
+        self,
+        rho,
+        stream_speed,
+        alpha,
+        beta,
+        mach,
+        reynolds,
+        pitch_rate,
+        yaw_rate,
+        roll_rate,
+    ):
```

Comment:
When I used all arguments as `0` (or null vectors/tuples, depending on the type) on the `GenericSurfaces` class, the function worked and returned `0` for all its return values. For this function here in `LinearGenericSurfaces`, I got a division by zero error. Is this behavior okay?